examples/js/loaders/ColladaLoader.js

```diff
@@ -18,9 +18,9 @@ THREE.ColladaLoader.prototype = {
 	load: function ( url, onLoad, onProgress, onError ) {
 
 		var scope = this;
-
-		var path = THREE.Loader.prototype.extractUrlBase( url );
-
+		
+                var path = scope.path === undefined ? THREE.Loader.prototype.extractUrlBase( url ) : scope.path; 
+		
```

Comment:
Sry for being so pedantic. But it seems you used soft tabs here. Please ensure correct code style by using hard tabs instead.

Reply:
don't worry about it, you are totaly right ;)
didn't pay attention to this one